package.json

```diff
@@ -2,41 +2,26 @@
   "name": "yeoman-generator-list",
   "description": "Server for the Yeoman generator list on yeoman.io/generators",
   "repository": "yeoman/yeoman-generator-list",
-  "keywords": [
-    "generator",
-    "github",
-    "keyword",
-    "list",
-    "npm",
-    "package",
-    "packages",
-    "server",
-    "yeoman"
-  ],
   "author": "Yeoman",
   "license": "BSD",
```

Comment:
It's basically equivalent. I wouldn't change the license just for the sake of it.